test/src/index.test.js

```diff
@@ -310,5 +406,22 @@ describe('Anonymizer', () => {
         });
       });
     });
+
+    describe.skip('benchmark', () => {
+      Object.entries(samples).forEach(([sample, data]) => {
+        it(`should run '${sample}' in less than '350' ms`, () => {
+          const anonymize = anonymizer({ blacklist: ['*'] });
```

Comment:
Could we use a serializer and a strategy here?

Reply:
Done